client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { includes } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { AUTOMATED_TRANSFER_STATUS } from 'state/automated-transfer/constants';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getAutomatedTransferStatus } from 'state/automated-transfer/selectors';
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+
+class PluginAutomatedTransfer extends Component {
+
+	static propTypes = {
+		plugin: PropTypes.object,
+		status: PropTypes.string,
+		translate: PropTypes.func,
+	}
+
+	static defaultProps = {
+		plugin: {
+			name: 'Plugin',
+		},
+	}
+
+	getNoticeText = ( pluginName = '' ) => {
+		const { status, translate } = this.props;
+		const { START, SETUP, LEAVING, CONFLICTS, COMPLETE } = AUTOMATED_TRANSFER_STATUS;
+
+		switch ( status ) {
+			case START: return translate( 'Installing %s…', { args: pluginName } );
```

Comment:
Instead of `%s`, calling it `%(plugin)s` would help translators understand what the placeholder is for